src/main/java/fr/insalyon/creatis/gasw/GaswInput.java

```diff
@@ -48,13 +48,27 @@ public class GaswInput {
     private List<GaswUpload> uploads;
     private Map<String, String> gaswVariables;
     private Map<String, String> envVariables;
-
+    private String invocationString;
+    private String jobId;
+    private String applicationName;
+    private static String sourceFilePath;
+    private List<URI> downloadFiles;
+    private String outputDirName;
+    private Boolean moteurlite = false;
     /**
      *
      * @param executableName Name of the executable file.
      * @param parameters List of parameters associated with the command.
      * @param downloads List of input files to be downloaded in the worker node.
      * @param uploads List of output files to be uploaded to a Storage Element.
+     * @param gaswVariables Map of GASW variables.
+     * @param envVariables Map of environment variables.
+     * @param invocationString String representation of the invocation
+     * @param jobId
+     * @param sourceFilePath
+     * @param applicationName
+     * @param outputDirName
+     * @param downloadFiles
```

Comment:
comments not on the good constructor

Reply:
updated comments for the appropriate constructors